src/video_core/texture_cache/surface_base.cpp

```diff
@@ -248,8 +248,14 @@ void SurfaceBaseImpl::FlushBuffer(Tegra::MemoryManager& memory_manager,
 
     // Use an extra temporal buffer
     auto& tmp_buffer = staging_cache.GetBuffer(1);
+    /// Special case for 3D Texture Segments
```

Comment:
```suggestion
    // Special case for 3D Texture Segments
```

Otherwise this causes a warning with doxygen (and clang with -Wdocumentation).